.github/workflows/monitor.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: tests
+
+on:
+  push:
+    paths:
+    - "monitor/**"
+
+jobs:
+  lint_and_test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: monitor
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
We may eventually want to reference a shared yaml file to avoid duplication... but I could also see the workflows diverging so, this seems fine